### PR TITLE
Add Haml lint

### PR DESCRIPTION
**Pull request by agrim123**
- Added haml_lint gem
- Added haml-lint.yml

In continuation with #1396 
Addresses #1395

### CODE REVIEW

**Comment by agrim123** [comment]
@Ana06 Getting this error:
```
warning: parser/current is loading parser/ruby22, which recognizes
warning: 2.2.6-compliant syntax, but you are running 2.2.3.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
```
How to deal with it?

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ before_script:
   - cp config/secrets.yml.example config/secrets.yml
   - mysql -u root -e 'create database osem_test;'
   - RAILS_ENV=test bundle exec rake db:migrate --trace
+  - gem install haml_lint
```
Comment (Ana06) [inline .travis.yml]: why do you need this? Travis is already running bundle install :confused:

File: .travis.yml [hunk 2]
```diff
@@ -23,6 +23,8 @@ before_script:
...
 script:
   - 'bundle exec rubocop -Dc .rubocop.yml'
+  - 'bundle exec haml-lint -c .haml-lint.yml app/views'
```
Comment (Ana06) [inline .travis.yml]: `-c .haml-lint.yml` shouldn't be needed as `.haml-lint.yml` is the default config file.

File: .travis.yml (hunk 1 above)
Comment (agrim123) [inline .travis.yml]: It was returning 
`haml-lint command not found`!

**Pull request merged**